components/org.wso2.carbon.identity.scim.provider/src/main/java/org/wso2/carbon/identity/scim/provider/auth/SCIMAuthConfigReader.java

```diff
@@ -91,4 +91,24 @@
         }
         return Collections.emptyList();
     }
+
+    /**
+     * Get the configuration value from identity.xml file.
+     *
+     * @return true or false
+     */
+    public boolean isGetAllDetailsEnabled() {
+
+        IdentityConfigParser identityConfig = IdentityConfigParser.getInstance();
+        OMElement scimElem = identityConfig.getConfigElement(SCIMProviderConstants.ELEMENT_NAME_SCIM);
```

Comment:
Shall we change the variable name scimElem?

Reply:
It is the <SCIM> element we are taking. Hence, we'll make it "scimElement".